README.md

```diff
@@ -110,11 +110,37 @@ Production Image: (application is copied to image as well, hence no need to use
 ```bash
 $ docker build --target production --tag todo-app:prod .
 $ docker run --env-file .env -p 5000:5000 todo-app:prod
-
 ```
 
 ## **Module-6**
 
 The architecture diagrams can be found in documentation folder. This folder contains 3 .drawio files (one each for context, component, and container diagrams), and 4 html files which correspond to the drawio files.   
 
 The .drawio files can be edited through online tool available at https://app.diagrams.net/
+
+## **Module-7**
+
+### Pre-requisites 
+Use following commands to create test image.
+
+```bash
+$ docker run todo-app:test tests/test_viewmodel.py
```

Comment:
I think you've copied the wrong command in here; this is one of your run commands (not caught below) but I think you meant to have a build command here

Reply:
I did ! Thank you for pointing that out - corrected.